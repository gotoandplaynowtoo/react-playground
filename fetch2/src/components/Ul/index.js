import styled from 'styled-components';

const Ul = styled.ul`
  list-style: none;
  padding: 0 4px;
`;

export default Ul;