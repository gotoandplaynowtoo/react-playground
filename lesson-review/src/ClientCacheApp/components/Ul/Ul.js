import React from 'react';
import styled from 'styled-components';
import PropTypes from 'prop-types';

const Ul = styled.ul`
  padding: 4;
`;

Ul.propTypes = {
  children: PropTypes.node
};

Ul.defaultProps = {
  children: <li>Sample item</li>
};

export default Ul;