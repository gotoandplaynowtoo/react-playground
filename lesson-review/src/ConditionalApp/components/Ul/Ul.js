import PropTypes from 'prop-types';
import styled from 'styled-components';

const Ul = styled.ul`
  list-style: none;
`;

Ul.propTypes = {
  children: PropTypes.node
};

export default Ul;