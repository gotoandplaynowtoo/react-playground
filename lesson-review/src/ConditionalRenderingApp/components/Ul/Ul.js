import styled from 'styled-components';
import PropTypes from 'prop-types';

const Ul = styled.ul`
  list-style: none;
`;

Ul.propTypes = {
  children: PropTypes.node
};

export default Ul;