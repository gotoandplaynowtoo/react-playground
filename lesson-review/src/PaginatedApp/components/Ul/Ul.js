import styled from 'styled-components';
import PropTypes from 'prop-types';

const Ul = styled.ul`
  list-style: none;
  padding: 4px;
`;

Ul.propTypes = {
  children: PropTypes.node
};

export default Ul;