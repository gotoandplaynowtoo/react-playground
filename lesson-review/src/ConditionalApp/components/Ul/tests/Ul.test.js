import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Ul from 'ConditionalApp/components/Ul';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Ul/>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('matches to snapshot', () => {
  const tree = renderer.create(<Ul/>).toJSON();
  expect(tree).toMatchSnapshot();
});