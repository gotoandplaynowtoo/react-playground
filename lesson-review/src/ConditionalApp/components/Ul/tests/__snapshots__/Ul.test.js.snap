// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches to snapshot 1`] = `
<ul
  className="sc-bdVaJa DDnfd"
/>
`;
